main.go

```diff
@@ -19,6 +19,8 @@ func main() {
 	var err error
 
 	// Initialize & parse flags
+	var config server.Config
+	flag.StringVar(&config.ConfigPath, "config", "", "path to .yaml file config")
```

Comment:
Once we have the path to the config file, we'd need to parse its yaml and populate a `Config` struct with its values.

Reply:
Awesome! made some changes to populate the ```Config``` struct with the yaml file acquired from the given path. At the moment the only field is NeverEvictRepos

```go
type Config struct {
	NeverEvictRepos []string `yaml:"never-evict-repos"`
}
```